### ISSUE TITLE
error with toolset types plugin

### ISSUE THREAD

**Issue by Amba-fr** [soflyy/breakdance-bugs#80]
`{"type":"INTERNAL_SERVER_ERROR","errorDetails":{"type":"Breakdance\\Lib\\Vendor\\Whoops\\Exception\\ErrorException","message":"\"continue\" targeting switch is equivalent to \"break\". Did you mean to use \"continue 2\"?","code":2,"file":"/var/***/***/***/wp-content/plugins/types/vendor/toolset/types/embedded/includes/wpml.php","line":646,"trace":[{"file":"/var/***/***/***/wp-content/plugins/types/vendor/toolset/types/embedded/includes/wpml.php","line":646,"function":null,"class":"Breakdance\\Lib\\Vendor\\Whoops\\Exception\\ErrorException","args":["\"continue\" targeting switch is equivalent to \"break\". Did you mean to use \"continue 2\"?"],"snippet":{"641":"                $default = wpcf_custom_taxonomies_default();\n","642":"                if ( $label == 'name' || $label == 'singular_name' ) {\n","643":"                \t$string_context = array( 'domain'  => 'Types-TAX' );\n","644":"\t                $string_context['context'] = $label === 'name' ? 'taxonomy general name' : 'taxonomy singular name';\n","645":"                    wpcf_translate_register_string( $string_context, false, $string );\n","646":"                    continue;\n","647":"                }\n","648":"                if ( isset( $default['labels'][$label] ) && $string == $default['labels'][$label] ) {\n","649":"                    wpcf_translate_register_string( 'Types-TAX', $label, $string );\n","650":"                } else {\n"},"startingLine":641},{"file":"/var/***/***/***/wp-content/plugins/types/vendor/toolset/types/embedded/bootstrap.php","line":214,"function":"handleError","class":"Breakdance\\Lib\\Vendor\\Whoops\\Run","args":[],"snippet":{"209":"\n","210":"    // Module manager\n","211":"    require_once WPCF_EMBEDDED_INC_ABSPATH . '/module-manager.php';\n","212":"\n","213":"    // WPML specific code\n","214":"    require_once WPCF_EMBEDDED_INC_ABSPATH . '/wpml.php';\n","215":"\n","216":"    // CRED specific code.\n","217":"    if ( defined( 'CRED_FE_VERSION' ) ) {\n","218":"        require_once WPCF_EMBEDDED_INC_ABSPATH . '/cred.php';\n"},"startingLine":209},{"file":"/var/***/***/***/wp-content/plugins/types/vendor/toolset/types/embedded/bootstrap.php","line":214,"function":"require_once","class":null,"args":[],"snippet":{"209":"\n","210":"    // Module manager\n","211":"    require_once WPCF_EMBEDDED_INC_ABSPATH . '/module-manager.php';\n","212":"\n","213":"    // WPML specific code\n","214":"    require_once WPCF_EMBEDDED_INC_ABSPATH . '/wpml.php';\n","215":"\n","216":"    // CRED specific code.\n","217":"    if ( defined( 'CRED_FE_VERSION' ) ) {\n","218":"        require_once WPCF_EMBEDDED_INC_ABSPATH . '/cred.php';\n"},"startingLine":209},{"file":"/var/***/***/***/wp-includes/class-wp-hook.php","line":307,"function":"wpcf_embedded_init","class":null,"args":[],"snippet":{"302":"\n","303":"\t\t\t\t// Avoid the array_slice() if possible.\n","304":"\t\t\t\tif ( 0 == $the_['accepted_args'] ) {\n","305":"\t\t\t\t\t$value = call_user_func( $the_['function'] );\n","306":"\t\t\t\t} elseif ( $the_['accepted_args'] >= $num_args ) {\n","307":"\t\t\t\t\t$value = call_user_func_array( $the_['function'], $args );\n","308":"\t\t\t\t} else {\n","309":"\t\t\t\t\t$value = call_user_func_array( $the_['function'], array_slice( $args, 0, (int) $the_['accepted_args'] ) );\n","310":"\t\t\t\t}\n","311":"\t\t\t}\n"},"startingLine":302},{"file":"/var/***/***/***/wp-includes/class-wp-hook.php","line":331,"function":"apply_filters","class":"WP_Hook","args":[],"snippet":{"326":"\t *\n","327":"\t * @param array $args Parameters to pass to the callback functions.\n","328":"\t */\n","329":"\tpublic function do_action( $args ) {\n","330":"\t\t$this->doing_action = true;\n","331":"\t\t$this->apply_filters( '', $args );\n","332":"\n","333":"\t\t// If there are recursive calls to the current action, we haven't finished it until we get to the last one.\n","334":"\t\tif ( ! $this->nesting_level ) {\n","335":"\t\t\t$this->doing_action = false;\n"},"startingLine":326},{"file":"/var/***/***/***/wp-includes/plugin.php","line":476,"function":"do_action","class":"WP_Hook","args":[],"snippet":{"471":"\t} elseif ( is_array( $arg[0] ) && 1 === count( $arg[0] ) && isset( $arg[0][0] ) && is_object( $arg[0][0] ) ) {\n","472":"\t\t// Backward compatibility for PHP4-style passing of `array( &$this )` as action `$arg`.\n","473":"\t\t$arg[0] = $arg[0][0];\n","474":"\t}\n","475":"\n","476":"\t$wp_filter[ $hook_name ]->do_action( $arg );\n","477":"\n","478":"\tarray_pop( $wp_current_filter );\n","479":"}\n","480":"\n"},"startingLine":471},{"file":"/var/***/***/***/wp-settings.php","line":598,"function":"do_action","class":null,"args":[],"snippet":{"593":" *\n","594":" * If you wish to plug an action once WP is loaded, use the {@see 'wp_loaded'} hook below.\n","595":" *\n","596":" * @since 1.5.0\n","597":" */\n","598":"do_action( 'init' );\n","599":"\n","600":"// Check site status.\n","601":"if ( is_multisite() ) {\n","602":"\t$file = ms_site_check();\n"},"startingLine":593},{"file":"/var/***/***/***/wp-config.php","line":112,"function":"require_once","class":null,"args":["/var/***/***/***/wp-settings.php"],"snippet":{"107":"/** Chemin absolu vers le dossier de WordPress. */\n","108":"if ( !defined('ABSPATH') )\n","109":"define('ABSPATH', dirname(__FILE__) . '/');\n","110":"\n","111":"/** Réglage des variables de WordPress et de ses fichiers inclus. */\n","112":"require_once(ABSPATH . 'wp-settings.php');"},"startingLine":107},{"file":"/var/***/***/***/wp-load.php","line":50,"function":"require_once","class":null,"args":["/var/***/***/***/wp-config.php"],"snippet":{"45":" * If neither set of conditions is true, initiate loading the setup process.\n","46":" */\n","47":"if ( file_exists( ABSPATH . 'wp-config.php' ) ) {\n","48":"\n","49":"\t/** The config file resides in ABSPATH */\n","50":"\trequire_once ABSPATH . 'wp-config.php';\n","51":"\n","52":"} elseif ( @file_exists( dirname( ABSPATH ) . '/wp-config.php' ) && ! @file_exists( dirname( ABSPATH ) . '/wp-settings.php' ) ) {\n","53":"\n","54":"\t/** The config file resides one level above ABSPATH but is not part of another installation */\n"},"startingLine":45},{"file":"/var/***/***/***/wp-admin/admin-ajax.php","line":22,"function":"require_once","class":null,"args":["/var/***/***/***/wp-load.php"],"snippet":{"17":"if ( ! defined( 'WP_ADMIN' ) ) {\n","18":"\tdefine( 'WP_ADMIN', true );\n","19":"}\n","20":"\n","21":"/** Load WordPress Bootstrap */\n","22":"require_once dirname( __DIR__ ) . '/wp-load.php';\n","23":"\n","24":"/** Allow for cross-domain requests (from the front end). */\n","25":"send_origin_headers();\n","26":"\n"},"startingLine":17}]},"requestDetails":{"friendlyName":"Get templates data","responseCode":"500","requestUrl":"https://www.amba.pro/wp-admin/admin-ajax.php","requestMethod":"POST","body":{"action":"breakdance_manage_templates_load_data","post_type":"breakdance_template"}}}`

**Comment by louisreingold** [comment]
We're not able to reproduce this. If you email support@breakdance.com with your WP admin login details we'd love to take a closer look.
